README.md

```diff
@@ -592,25 +758,59 @@ curl -iX POST \
 }
 ```
 
-### List N Sampled Values at an Offset
+### :five: List N Sampled Values at an Offset
+
+This example shows the fourth, fifth and sixth sampled `count` values of `Motion:001`.
+
+#### Using QuantumLeap API
```

Comment:
It would make sense to re-order the tutorial the set of Queries into:

* A set of queries using the QuantumLeap API
* A second set of queries using CrateDB API

The switching between the two doesn't help the flow of the learning material

Yes I know the result is the same, but the **Use Case** for each is different. You should introduce and promote the QuantumLeap API (porcellan) above the CrateDB API (plumbing) as you should only need the latter in exceptional circumstances

Reply:
Agree!